enterprise/api/v1/urls.py

```diff
@@ -7,16 +7,15 @@
 
 from enterprise.api.v1 import views
 
-# Pylint considers module level variable as "Constant" and expects them to be upper-cased
-# that is why we have disabled 'invalid-name' check for variable definition below.
 router = DefaultRouter()  # pylint: disable=invalid-name
 router.register("site", views.SiteViewSet, 'site')
 router.register("auth-user", views.UserViewSet, 'auth-user')
-router.register(
-    "enterprise-course-enrollment",
-    views.EnterpriseCourseEnrollmentViewSet,
-    'enterprise-course-enrollment'
-)
+router.register("enterprise-catalog", views.EnterpriseCustomerCatalogViewSet, 'enterprise-catalog')
+router.register("enterprise_catalogs", views.EnterpriseCustomerCatalogApiViewSet, 'enterprise_catalogs')
```

Comment:
@UmanShahzad OK, but I'm still wondering if we can make these routes consistent? That is, if underscores are preferred, why not use them in L13 above?